tests/test_shuffled_indices.cpp

```diff
@@ -50,23 +50,29 @@ int main(int argc, char *argv[]) {
 
   try {
     // Initialize options db (this parses the command line)
-    options *opts = options::get();
-    opts->init(argc, argv);
-    if (opts->has_string("h") or opts->has_string("help") or argc == 1) {
-      print_help(*comm);
+    auto& arg_parser = global_argument_parser();
+    construct_all_options();
+		arg_parser.add_flag(FN,
```

Comment:
```suggestion
		arg_parser.add_flag("fn",
```

Reply:
@benson31 Is there a reason that this one doesn't use the FN macro, where the other tests use it?